### PR TITLE
logical_xor

### LINKED ISSUE #22990: logical_xor

### PR DESCRIPTION
**Pull request by roudik**
# PR Description 

## Related Issue 

Close #22990

## Checklist 

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by roudik** [comment]
Hey @bipinKrishnan. Are there any issues with this request?

**Comment by bipinKrishnan** [comment]
Hey @roudik , thanks for the contribution! 
It would be great if you could install pre-commit in your system so that the lint tests doesn't fail 🙂

**Comment by bipinKrishnan** [comment]
ivy-gardener

**Comment by roudik** [comment]
> Hey @roudik , thanks for the contribution! It would be great if you could install pre-commit in your system so that the lint tests doesn't fail 🙂

Hello, is this a requirement or can we close this PR now? (This is for a job application to Ivy)

**Comment by roudik** [comment]
Dear. @bipinKrishnan, confused about whether there is still anything to be fixed as the tests show as passed. We should be able to close this PR then?

**Comment by roudik** [comment]
@bipinKrishnan ??

File: ivy/functional/frontends/torch/tensor.py [hunk 1]
```diff
@@ -437,6 +437,10 @@ def logical_not_(self):
     def logical_or(self, other):
         return torch_frontend.logical_or(self, other)
 
+    @with_unsupported_dtypes({"2.0.1 and below": ("bfloat16",)}, "torch")
```
Comment (bipinKrishnan) [inline ivy/functional/frontends/torch/tensor.py:440]: Seems like the tests are failing due to an unsupported dtype not listed here. You could check the tests and add the unsupported dtype here.

Comment (roudik) [reply ivy/functional/frontends/torch/tensor.py:440]: Which tests are failing? To me it shows that only run_test (1) is failing which should be ignored as per the instructions above?

Comment (roudik) [reply ivy/functional/frontends/torch/tensor.py:440]: Not sure what you mean by unsupported dtype not listed, do you mean to remove the line 440? where do you want me to add unsupported dtype?

**Comment by roudik** [comment]
> Thanks for contributing to Ivy! 😊👏 Here are some of the important points from our Contributing Guidelines 📝: 1. Feel free to ignore the `run_tests (1)`, `run_tests (2)`, … jobs, and only look at the `display_test_results` job. 👀 It contains the following two sections: - **Combined Test Results:** This shows the results of all the ivy tests that ran on the PR. ✔️ - **New Failures Introduced:** This lists the tests that are passing on main, but fail on the PR Fork. Please try to make sure that there are no such tests. 💪 2. The `lint / Check formatting / check-formatting` tests check for the formatting of your code. 📜 If it fails, please check the exact error message in the logs and fix the same. ⚠️🔧 3. Finally, the `test-docstrings / run-docstring-tests` check for the changes made in docstrings of the functions. This may be skipped, as well. 📚 Happy coding! 🎉👨‍💻

@bipinKrishnan Doesn't I say above that we can feel free to ignore run_tests?

**Comment by bipinKrishnan** [comment]
Hey @roudik, its not like that actually. The display_test_results checks for newly introduced failures, so if you open it and check "New failures introduced" you can see that the tests you added are failing. If you need to look into the details of why its failing, you can check "run_tests (1)" which contains the detailed output/traceback of your test. Hope that makes sense.

**Comment by roudik** [comment]
> Hey @roudik, its not like that actually. If you read carefully, you will see that "display_test_results" checks for newly introduced failures, so if you open it and check "New failures introduced" you can see that the tests you added are failing. If you need to look into the details of why its failing, you can check "run_tests (1)" which contains the detailed output/traceback of your test. Hope that makes sense.

Dear @bipinKrishnan, now that makes sense, but I am not quite sure about what to add to the unsupported types. The working functions loical_or and logical_and both just use bfloat16, and logical_not doesn't even include the line about unsupported types. I have noticed about the failed test mentioning about uint16, but when I tried adding it to the unsupported_dtypes line [...("bfloat16", "uint16",)...] I got even more errors than before.

**Pull request closed without merging**